lnetatmo.py

```diff
@@ -137,6 +139,15 @@ def __init__(self, authData):
             print("lnetatmo.py : warning, no camera available for testing")
 
     try:
+        HomeCoach = HomeCoachData(authorization)
+        print(HomeCoach)
+    except NoDevice :
+        if stdout.isatty():
+            print("lnetatmo.py : warning, no camera available for testing")
```

Comment:
Wrong copy/paste message should be: "lnetatmo.py : warning, no HomeCoach available for testing"